pandas/core/indexes/base.py

```diff
@@ -695,13 +695,31 @@ def memory_usage(self, deep=False):
     # ops compat
     @deprecate_kwarg(old_arg_name='n', new_arg_name='repeats')
     def repeat(self, repeats, *args, **kwargs):
-        """
-        Repeat elements of an Index. Refer to `numpy.ndarray.repeat`
-        for more information about the `repeats` argument.
+        """Repeat elements of an Index.
+
+        Parameters
+        ----------
+        repeats : int
+            The number of repetitions for each element.
+
+        Returns
+        -------
+        pandas.Index
+            Newly created Index with repeated elements.
 
         See also
         --------
-        numpy.ndarray.repeat
+        numpy.repeat
```

Comment:
Some information on why `numpy.repeat` is in the See Also, and differences among them if they exist would be useful.

Also, that's more an open question, but could make sense to list here something like `Index.is_unique` for example? Probably if we describe better real use-cases, we'll find other related methods.